src/views/NoteEditView.vue

```diff
@@ -1,7 +1,366 @@
 <template>
-  <h1>Note Edit</h1>
+  <div class="note-edit-view">
+    <div class="menu">
+      <div class="note-info-container">
+        <div class="note-info">
+          <label for="channel">channel</label>
+          <input
+            type="text"
+            name="channel"
+            id="channel"
+            placeholder="#event/hackathon/25spring/16"
+            v-model="noteChannel"
+            :disabled="notesStore.loading"
+          />
+        </div>
+        <!-- 必要かどうか、後で検討する -->
+        <!-- <div class="note-info">
+          <label for="tags">tags</label>
+          <input
+            type="text"
+            name="tags"
+            id="tags"
+            placeholder="tag1,tag2,tag3,tagtag"
+            v-model="noteTags"
+            :disabled="notesStore.loading"
+          />
+        </div> -->
+      </div>
+      <div class="button-container">
+        <button>キャンセル</button>
+        <button>保存</button>
+      </div>
+    </div>
+    <div class="editor-container">
+      <div class="editor-pane">
+        <div v-if="notesStore.loading" class="loading-overlay">
+          <div class="loading-message">読み込み中...</div>
+        </div>
+        <div class="editor-content">
+          <textarea
+            placeholder="このノートは、Markdown 形式で入力できます。traQ のチャンネルと紐づけることで、ノートを簡単に管理できます。"
+            v-model="noteBody"
+            :disabled="notesStore.loading"
+          ></textarea>
+        </div>
+      </div>
+      <div class="preview-pane">
+        <div class="preview-content">
+          <h1>Lorem ipsum</h1>
+          <p>
+            dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore
+            et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+            laboris nisi ut aliquip ex ea commodo consequat.
+          </p>
+
+          <h2>サブヘッダー</h2>
+
+          <p><strong>太字のテキスト</strong>と<em>斜体のテキスト</em>の例です。</p>
+
+          <ul>
+            <li>リストアイテム1</li>
+            <li>リストアイテム2</li>
+            <li>リストアイテム3</li>
+          </ul>
+
+          <ol>
+            <li>番号付きリスト1</li>
+            <li>番号付きリスト2</li>
+            <li>番号付きリスト3</li>
+          </ol>
+
+          <p><code>インラインコード</code>の例です。</p>
+
+          <pre><code class="language-javascript">// コードブロックの例
+function hello() {
+  console.log("Hello, World!");
+}</code></pre>
+
+          <blockquote>
+            <p>
+              これは引用文です。<br />
+              複数行にわたって書くことができます。
+            </p>
+          </blockquote>
+
+          <p><a href="https://example.com">リンクの例</a></p>
+        </div>
```

Comment:
後述する レンダリングされた markdown をここで表示する形にしましょう

```suggestion
        <div class="preview-content" v-html="renderedMarkdown"></div>
```